src/CheckoutSdk/Marketplace/Payout/Response/GetScheduleResponse.cs

```diff
@@ -0,0 +1,10 @@
+using Checkout.Common;
+using System.Collections.Generic;
+
+namespace Checkout.Marketplace.Payout.Response
+{
+    public class GetScheduleResponse : Resource
+    {
+        public Dictionary<Currency, CurrencySchedule> Currency { get; set; }
```

Comment:
What do you think about initializing here the map? I think it's more safe to return an empty map instead of a null reference. 
`public Dictionary<Currency, CurrencySchedule> Currency { get; set; } = new Dictionary<Currency, CurrencySchedule>();`